utils/fixed_decimal/src/decimal.rs

```diff
@@ -178,6 +178,26 @@ impl FixedDecimal {
         Ok(result)
     }
 
+    /// Initialize a `FixedDecimal` with a floating-point value, using Ryu under the hood
+    /// to get digits from the value.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use fixed_decimal::FixedDecimal;
+    ///
+    /// let dec = FixedDecimal::from_float_ryu::<f32>(12.34).unwrap();
+    /// assert_eq!(4, dec.digit_at(-2));
+    /// assert_eq!(3, dec.digit_at(-1));
+    /// assert_eq!(2, dec.digit_at(0));
+    /// assert_eq!(1, dec.digit_at(1));
+    /// ```
+    #[cfg(feature = "ryu_decimal")]
+    pub fn from_float_ryu<F: ryu::Float>(value: f64) -> Result<Self, Error> {
```

Comment:
Please add another argument here for the number of decimal places, perhaps an i16.  For example, "1.2" and "1.20" are different FixedDecimal values, but `f64` doesn't have a way to distinguish them.

Reply:
Resolved! Had to switch to a fork of the library due to https://github.com/dtolnay/ryu/issues/27. Also introduced a bit of logic duplication, but not sure if the abstractions needed to eliminate the duplication are worth it.